pkg/config/plugins/resources/postgres/config.go

```diff
@@ -25,6 +30,8 @@ type PostgresStoreConfig struct {
 	Password string `json:"password" envconfig:"kuma_store_postgres_password"`
 	// Database name of the Postgres DB
 	DbName string `json:"dbName" envconfig:"kuma_store_postgres_db_name"`
+	// Driver to use, one of: pgx, postgres
+	DriverName string `json:"driverName" envconfig:"kuma_store_postgres_driver_name"`
```

Comment:
Why is the driver configurable?

Reply:
I'd prefer this to be a flag for the next version. 2.2 is a candidate for LTS. I'm also extra cautious because of this.

I'd remove the flag in next+1. I'm slightly worried about a scenario where something will be wrong with the new implem/driver and the user will have no choice but roll back.

Reply:
I'm resolving this, @lahabana if you want this modified let's chat during the weekly meeting.